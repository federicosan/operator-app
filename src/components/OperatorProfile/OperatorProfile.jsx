import React, { Component, PropTypes } from 'react';
import { connect } from 'react-redux';

export class OperatorProfileComponent extends Component {
  render () {
    return (
      <div className="row profile" />
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({
  dispatch,
});

const OperatorProfile = connect(
  mapStateToProps,
  mapDispatchToProps,
)(OperatorProfileComponent);

export default OperatorProfile;
