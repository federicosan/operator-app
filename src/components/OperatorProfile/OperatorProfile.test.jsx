import React from 'react';

import { shallow } from 'enzyme';

import OperatorProfile from './OperatorProfile.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('OperatorProfile', () => {
  it('matches snapshot', () => {
    const component = shallow(<OperatorProfile store={store} />);

    expect(component).toMatchSnapshot();
  });
});
