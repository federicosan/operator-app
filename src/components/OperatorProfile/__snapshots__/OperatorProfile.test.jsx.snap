exports[`OperatorProfile matches snapshot 1`] = `
<OperatorProfileComponent
  dispatch={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
